src/ast/mod.rs

```diff
@@ -961,10 +981,8 @@ pub enum GenericAction<Head, Leaf, Ann> {
         Vec<GenericExpr<Head, Leaf, Ann>>,
         GenericExpr<Head, Leaf, Ann>,
     ),
-    /// `delete` an entry from a function.
-    /// Be wary! Only delete entries that are subsumed in some way or
-    /// guaranteed to be not useful.
-    Delete(Ann, Head, Vec<GenericExpr<Head, Leaf, Ann>>),
+    /// Subsume or delete an entry from a function.
```

Comment:
optional: leave a note that `delete` is more dangerous than `subsume`?

Reply:
I updated the docs to make it more explicit what subsume does. There is also a longer docstring on the `Change` enum above.